spec/features/admin_can_delete_an_item_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.feature "Admin can delete an item" do
+  scenario "Admin can delete an existing item and is redirected to items index" do
+    admin = User.create(first_name: "mister",
+                        last_name: "admin",
+                        username: "admin",
+                        password: "admin",
+                        role: 1)
+    travesty = Travesty.create(title: "Environmental Disasters")
+    item = travesty.items.create(title: "Arsenicify Water Sources",
+                       description: "There will be no investigation, cause they'll all be dead!",
+                       price: 10,
+                       image_url: "http://www.survivalreadyblog.com/uploads/1/4/1/2/1412634/7530107_orig.jpg")
```

Comment:
Line is too long. [110/80]<br>Align the elements of a hash literal if they span more than one line.